test/functional/shared.js

```diff
@@ -196,10 +197,26 @@ function MultisigScenario (size, amount) {
 	this.amount = amount || 100000000000;
 }
 
+function validateParameterResponse (res, paramNames, message) {
+	node.expect(res.body).to.have.property('message').to.equal(message);
+}
+
+function validateSwaggerParameterResponse (res, paramNames, message) {
```

Comment:
I suggest to change a name for sth like `expectSwaggerErrorResponse`

Reply:
agree with this naming